### PR TITLE
quick-start feedback

### PR DESCRIPTION
**Pull request by janainawittner**
A few corrections and [Comments] within the text. 
I let you review, make the needed changes.

Additional feedback: 

If I understand, you are suggesting here to send request from an inner system. Is that as efficient as using a complex menu on LivePepper to send orders and discounts?

### CODE REVIEW

**Comment by janainawittner** [comment]
@antmonnier, @atorin, I see that some answers to my questions are in the **Integration Guide**. Don't be too quick goiong over my comments, I will need to edit them once I finish to read the Integration Guide. I will let you know when I am done. It's looking great by the way. Exactly what I keep writing over and over on email. I love it!

**Comment by janainawittner** [comment]
@antmonnier, @atorin, I am done. There is almost nothing left here :-)

File: content/developers/en/quick-start.md [hunk 1]
```diff
@@ -63,10 +65,12 @@ https://manager.hubrise.com/oauth2/v1/authorize?
 You will have the chance to review the connection details and to grant access permissions to your application.
 
 If everything goes well, you should see a page similar to the following.
+[comment]: # (I THINK IT WOULD BE BETTER TO GIVE THIS PAGE A NAME, AND INCLUDE THE SCREENSHOT BELOW)
```
Comment (atorin) [inline content/developers/en/quick-start.md]: @janainawittner Thanks for pointing this out. It's very bad practice to refer to an image without explaining what it contains, indeed... I've rephrased this group of sentences to hopefully make things clearer.

**Comment by atorin** [comment]
@janainawittner I've addressed your comments, spot on as usual! If you're happy with the changes @antmonnier can merge.

Regarding your other comment:
>If I understand, you are suggesting here to send request from an inner system. Is that as efficient as using a complex menu on LivePepper to send orders and discounts?

Yes, they are equivalent approaches. We just didn't want to mention any particular 3rd party solution in this quick start, but one could easily connect LP and send test orders from there.

**Comment by janainawittner** [comment]
@atorin, thank you for the prompt corrections :-)
@antoine , it's all yours.

**Pull request merged**